tests/MongoDB.Driver.Tests/Specifications/command-monitoring/tests/find.json

```diff
@@ -102,7 +102,6 @@
             },
             "$returnKey": false,
             "$showDiskLoc": false,
-            "$snapshot": false
```

Comment:
Latest versions of the server return an error if the command includes $snapshot (even if set to false).